packages/meditrak-server/src/routes/authenticate.js

```diff
@@ -57,6 +57,10 @@ const getAuthorizationObject = async ({ accessPolicy, refreshToken, user, permis
   const userDetails = {
     id: user.id,
     name: user.fullName,
+    firstName: user.first_name,
+    lastName: user.last_name,
+    position: user.position,
+    employer: user.employer,
```

Comment:
This adds the extra fields to every client doing authentication. I think I'd prefer to add a `GET /me` endpoint that just returns more info about the user attached to the auth header

Reply:
hmmm I feel like there is no real downside to including a few extra fields on each response and it saves doing an extra request every time the user logs in. Plus if we keep using `name` in this endpoint and then have another endpoint with `firstName` and `lastName`, we would have to be careful not to let them get out of sync in an app. eg. user details have been changed in AdminPanel and PSSS user has not logged back out but has been re-authenticated in the background.

Reply:
Fair enough, I'm happy with that :-)